### PR TITLE
FEATURE Use geo api

### PR DESCRIPTION
**Pull request by DanielSiepmann**
This pull request includes a New feature
The following changes were made

* Track user location
* Update map position and marker
* Send request to scan surrounding area

### CODE REVIEW

**Comment by JakeCooper** [comment]
Dupe of PR #725

**Comment by megahambone** [comment]
It'll have the bug my first version did in that every 10 seconds it'll rehome the map to your location, so if you zoom, or move the map to look at a farther off pokemon, it'll annoy the hell out of you.

**Comment by JakeCooper** [comment]
Ohh I remember resolving that issue last time I was working with Maps. I think you can just set an option to not snap to the waypoint.

**Comment by megahambone** [comment]
If you're using panTo, I don't think that works. I do have to admit, it looks cleaner, but I know it has some bugs I've already resolved in #725

File: static/map.js [hunk 1]
```diff
@@ -399,3 +399,35 @@ var updateLabelDiffTime = function() {
...
+                map.panTo(googleLatLang);
+                marker.setMap(null);
+                marker = new google.maps.Marker({
+                    position: googleLatLang,
+                    map: map,
+                    animation: google.maps.Animation.DROP
+                });
+            });
+        };
+        window.setInterval(updateLocation, 10 * 1000);
```
Comment (MichaelErmer) [inline static/map.js]: I recommend you do a `setTimeout(updateLocation, 10 * 1000);` at the end of updateLocation, so if it takes a little longer its assured that only one updateLocation runs at a time.

**Comment by MichaelErmer** [comment]
Please add a toggle to the menu to enable this feature, if you share the map with a group of people  only one should be the "master" to prevent location jumps.

**Comment by DanielSiepmann** [comment]
Will solve #1002

**Comment by pickett2k** [comment]
how do i install this please?

**Comment by megahambone** [comment]
#725 has the features above plus the enhancement of being able to pan the map without it recentering on every location update and the ability to turn off geolocation entirely.

**Pull request closed without merging**